validator.proto

```diff
@@ -11,7 +11,7 @@ package validator;
 
 import "google/protobuf/descriptor.proto";
 
-option go_package = "validator";
+option go_package = "github.com/mwitkow/go-proto-validators/validator";
```

Comment:
I think it has to be a `;` instead of `/` before validator (as the warning suggests)

Reply:
The doc mentions this:
`The Go package name of generated code will be the last path component of the go_package option.`
If I understood it correctly the `;...` is optional. And if it is omitted the last part of the path will be used. Thus changing this to `.../validator;validator` will not change anything, currently it does not give warnings anymore. However, since it will be really easy to add I'd be happy to if your're still getting warnings :)

Reply:
still getting the same warning with PROTOC_VERSION=3.11.4

Reply:
@doriandekoning I might be wrong, but in my understanding, the first part should be the Go import path, which in this case is `github.com/mwitkow/go-proto-validators/`. Then the second part after the `;` is for the package name (this is just a guess because the documentation doesn't mention anything about the `;` meaning in this context).
I'm not getting a warning with your change, but I'm getting a compile error on my project, which disappears if I use `github.com/mwitkow/go-proto-validators;validator` as suggested by the warning. (Note also that it's not `.../validator;validator` but `.../go-proto-validators;validator`).

Reply:
@tTibo-lg We're not getting the compilation error in our project but I'll change it to `...go-proto-validators;validator` in that case :)

Reply:
It should be resolved now!

Reply:
nit: it seems you changed to `.../validator;validator` instead of `.../go-proto-validators;validator`

Reply:
Woops, how have I missed that...? Now I also understand why it would give an error in some cases.  Anyway, it should be fixed now